pyproject.toml

```diff
@@ -0,0 +1,157 @@
+[build-system]
+requires = ["poetry-core>=1.0.0,<2.0", "poetry-dynamic-versioning>=1.0.0,<2.0.0"]
+build-backend = "poetry_dynamic_versioning.backend"
+
+[tool.poetry]
+name = "imap-processing"
+# Gets updated dynamically by the poetry-dynamic-versioning plugin
+version = "0.0.0"
+description = "IMAP Science Operations Center Processing"
```

Comment:
You will want to update this to reflect the description of this repo, instead of the `imap_processing` repo